contrib/mmgrok/mmgrok.c

```diff
@@ -284,33 +284,33 @@ parse_result_store(const grok_match_t gm,instanceData *pData)
 	RETiRet;
 }
 
-/* motify message for per line */
+/* notify message for per line */
```

Comment:
Thinking about this a bit more, I suspect it's probably `modify` not `notify`

Reply:
I've moved this to https://github.com/rsyslog/rsyslog/commit/ebca5bc0327c53375ff8b833aefeac1d1674d06c